raiden-ts/docs-source/private-chain.md

```diff
@@ -1,68 +1,86 @@
+  - [Locate the smart contract version](#locate-the-smart-contract-version)
 - [Setting up the Development Environment](#setting-up-the-development-environment)
-  * [Locating the Smart Contracts](#locating-the-smart-contracts)
-  * [Starting the SDK](#starting-the-sdk)
-  * [Changing the dApp to work on the private setup](#changing-the-dapp-to-work-on-the-private-setup)
+  - [Locating the Smart Contracts](#locating-the-smart-contracts)
+  - [Starting the SDK](#starting-the-sdk)
+  - [Changing the dApp to work on the private setup](#changing-the-dapp-to-work-on-the-private-setup)
 
 ## Preface
+
 This guide targets users that want to use the Light Client SDK or dApp with a custom network setup. This includes private chain deployments of the Raiden Network or unsupported networks (e.g., Kovan).
 
 To use a private network you first have to deploy the contracts by hand.
 Start by following this tutorial on how to deploy and run Raiden Network on a private chain:
-- [Raiden on Private Network Tutorial] 
 
-After finishing the guide above you should have a running Raiden node along with a private chain with the Raiden Network smart contracts deployed. Now you can move on to setting up the Light Client on your private network.
+- [Raiden on Private Network Tutorial]
+
+After finishing the guide above you should have a running Raiden node along with a private chain with the Raiden Network smart contracts deployed. But before that please go through **Setting up the Light Client on a private network** just below to figure the correct version of `raiden-contracts` and subsequently of `raiden` which will help us deploy raiden-contracts correctly.
 
 ## Setting up the Light Client on a private network
+
 ### Identifying the correct smart contract version
+
 Before starting your private network setup, you first need to:
+
 1. Select which version of the Light Client you want to use
 2. Locate the version of the smart contracts used in that specific version of the Light Client
 
 ### Locate the smart contract version
 
 You can easily find out the version of the smart contracts used in your version of the Light Client by checking the `raiden-contracts` submodule in the Light Client GitHub repo.
+
 1. Navigate to the [GitHub] project page
-2. Locate the `raiden-ts` subdirectory and look for the `raiden-contracts` submodule *(something like`raiden-contracts @ dfc0da6`)*
-3. Look at the commit message and check if it includes the `raiden-contracts` version. If not, follow the submodule link to the contracts repository and locate the `setup.py` file. You'll find the version of the Raiden Contracts by looking at the version variable *(`VERSION = '0.33.3'`)*
+2. Locate the `raiden-ts` subdirectory and look for the `raiden-contracts` submodule _(something like`raiden-contracts @ 770c8cd`)_
+   ![raiden-contracts](https://user-images.githubusercontent.com/15123108/127455881-16d61132-1ded-4299-99ee-6a87298408be.png)
```

Comment:
I don't like having screenshots in docs like this, they are hard to update and (in my opinion) harder to make sense of than well written text.

@taleldayekh opinion?

Reply:
I agree, whenever we've used screenshots they've quickly gotten outdated. Better to just use text.

Reply:
IMO screenshots can be useful, but not like this; in the perfect world, we could gather screenshots directly from cypress run and update them from CI; maybe this is a nice-to-have feature for the future.

Reply:
Ok I will remove this image from here, as we a majority for that.

Reply:
@palango I have removed all the images from  the guide which i had put.